client/src/router/index.ts

```diff
@@ -14,6 +14,11 @@ const routes: Array<RouteRecordRaw> = [
     name: 'home',
     component: () => import('@/views/home/HomePage.vue'),
   },
+  {
+    path: '/recoveryImport',
+    name: 'recoveryImport',
+    component: () => import('@/views/devices/ImportRecoveryDevicePage.vue'),
+  },
```

Comment:
recoveryImport should not be an independent page, since it has some elements in common with HomePage, it should:
 - be a sub-part of HomePage that you show on condition
 OR
 - be a child route of HomePage, that would show the page based on the same system the logged in pages use currently.